plugins/modules/zos_data_set.py

```diff
@@ -247,6 +247,18 @@
         that is not available, then the value C(TMPHLQ) is used.
     required: false
     type: str
+  force:
+    description:
+      - Specifies that the data set can be shared with others during a member
+        delete operation which results in the data set you are updating to be
+        simultaneously updated by others.
+      - This is helpful when a data set is being used in a long running process
+        such as a started task and you are wanting to update or read.
+      - The C(-f) option enables sharing of data sets through the disposition
```

Comment:
I did fix the zos_blockfile doc a day ago, this should read:
```
      - The I(force=true) option enables sharing of data sets through the disposition
         I(DISP=SHR).
      - The I(force=true) only applies to data set members when I(state=absent) and
         I(type=MEMBER).
```

I added an extra line to highlight even more this support is just for data set members